app/forms/idv/doc_pii_form.rb

```diff
@@ -0,0 +1,46 @@
+module Idv
+  class DocPiiForm
+    include ActiveModel::Model
+
+    validate :validate_pii
```

Comment:
A heads up that I see this eventually growing into more validations. I just got an email from SBA about an error that occurs if they upload a test doc with an invalid state or invalid date. We may be adding validations for those as well in the near future.